manim/mobject/text/text_mobject.py

```diff
@@ -406,7 +406,7 @@ def __init__(
         text: str,
         fill_opacity: float = 1.0,
         stroke_width: float = 0,
-        color: Color | None = None,
+        color: str | None = None,
```

Comment:
The type hint should still include `Color`.

```suggestion
        color: Color | str | None = None,
```